verifier/testreport.py

```diff
@@ -198,29 +139,57 @@ def summary_status(self):
     return self.tests_fail == 0 and not self.missing_verify_data
 
   def compute_category_summary(self, items, group_tag, detail_tag):
```

Comment:
Rename `items` parameter to `test_errors`, or maybe something more generic if it's errors + unsupported exceptions + failures...
Rename function name to have the word "error" in it to be more descriptive/precise.